### PR TITLE
Change f2 to be exch instead of #

### PR DESCRIPTION
**Pull request by tekenny**
This PR is for issue #58.

I've included the VCL files that I needed to be able to build and execute MR via Delphi IDE.

Please let me know if you have any questions or concerns.

### CODE REVIEW

File: .github/DEVELOPERS.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+1 - Fork the repo to your own GitHub account.
+
+1 - Get your IDE (Integrated Development Environment) setup
+This project is currently supported with the use of two seperate IDEs for Pascal
+- Lazarus version 2.2.4 (fpc 3.2.2)
+Lazarus was the original IDE used for MorseRunner
+- Delphi Community Edition (aka RX RAD Studio 10.4)
+Delphia CE is free for use by open source projects and Delphi is the perferred IDE at this time
+Please not that there are github branches to support each of these IDEs
```
Comment (w7sst) [inline .github/DEVELOPERS.md]: should be Please 'note' that

**Review by w7sst — APPROVED** [review]

**Comment by w7sst** [comment]
Hi Tom, @tekenny
So what is your recommended process here... You have submitted a PR to our main branch and I have reviewed it. Do I perform the merge, or do you perform the merge? What process do you recommend on who performs the merge on a project like this?

I'm new to open source projects, so I'm open to best practices and recommendations. I just realized that when you submit PR, the action is probably back on me to do the merge. Just wondering what your thoughts are on this one.

We should probably document this process in our [Pull Requests](https://github.com/w7sst/MorseRunner/blob/main/.github/CONTRIBUTING.md#pull-requests) section of our [CONTRIBUTING.md page](https://github.com/w7sst/MorseRunner/blob/main/.github/CONTRIBUTING.md).

**Review by ct7aup — APPROVED** [review]

**Comment by tekenny** [comment]
Sorry for the delay, it has been very busy the last couple of days.

My experience with open source projects has been that anybody that wants to contribute forks the (upstream) repo.
They make a branch in their forked repo for the change and then create a pull request from the branch in their repo to the upstream repo.
The upstream repo protects merges to one or a very limited set of moderators to keep control on accepting consistentancy and quality.
In my opinion since we are small you @w7sst would be the person performing merges as anything accepted should match your vision for this repo. If the set of contributors grows greatly you may want to have others help.
IMHO if you let any of us to do merges choas could happen. Realistically this probably won't happen right now, since there is allot of discussions about the issues and changes being made and why we are doing things in the way we are doing them. But it is always possible that at one point a contributor could join that isn't as communitive, or wants things their way and that could disrupt things.

I hope this helps. Mike the decision is yours and I'll abide by it. Of course let me know if you have any questions or concerns with what I've stated. I'll be happy to create an issue to update Pull Requests section in CONTRIBUTING.md once you decide how you want to manage this repo.

**Comment by tekenny** [comment]
PS - I'll try to find a web page to reference as well regarding what I've stated. Meaning summary in CONTRIBUTING doc but also a link to a more extensive explanation to support the summary...

**Review by ct7aup — APPROVED** [review]

**Comment by w7sst** [comment]
> I hope this helps. Mike the decision is yours and I'll abide by it. Of course let me know if you have any questions or concerns with what I've stated. I'll be happy to create an issue to update Pull Requests section in CONTRIBUTING.md once you decide how you want to manage this repo.

Tom @tekenny, yes this helps and confirms my role. Like I said, this is my first time contributing to an open source project and I'm also trying to grow a community to support MR. This is starting slow and it starting to gain steam - I'm encouraged. I'll look into collaborator permissions to see if we can control who can perform the merges. I like the idea of keeping control of the merges so we can keep the project heading in a consistent direction.

**Comment by tekenny** [comment]
@w7sst no worries, I'm happy to help. Let me know if you need any help with admin of the repo. Also feel free to merge this PR.

**Comment by w7sst** [comment]
Merge complete.

**Pull request merged**